__tests__/index.js

```diff
@@ -226,10 +235,22 @@ describe('Manipulate', () => {
 
   it('Subtract', () => {
     const m = moment(time).subtract(7, 'days');
+    const n = new Date(
```

Comment:
I'd rather do something like `Date(time).getTime() - /* whatever in milliseconds */`

Reply:
Thanks, replaced